include/swift/Runtime/Concurrency.h

```diff
@@ -663,11 +663,23 @@ void swift_task_localValuePop();
 /// Its Swift signature is
 ///
 /// \code
-/// func _taskLocalValueGet<Key>(AsyncTask* task)
+/// func swift_task_localsCopyTo<Key>(AsyncTask* task)
 /// \endcode
 SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
 void swift_task_localsCopyTo(AsyncTask* target);
 
+/// Some task local bindings must be copied defensively when a child task is
+/// created in a task group. See task creation (swift_task_create_common) for
+/// a detailed discussion how and when this is used.
+///
+/// Its Swift signature is
+///
+/// \code
+/// func swift_task_localsCopyToTaskGroupChildTaskDefensively<Key>(AsyncTask* task)
+/// \endcode
+SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
+void swift_task_localsCopyToTaskGroupChildTaskDefensively(AsyncTask* target);
```

Comment:
I don't think this needs to be an exported runtime function.

Reply:
I think we can managed without exposing it -- will remove (and remove _before_ getting this into 6.0)

Reply:
removed in https://github.com/apple/swift/pull/74089